### ISSUE TITLE
Integrate with Choco vNext

### ISSUE THREAD

**Issue by RichiCoder1** [chocolatey/ChocolateyGUI#214]
Deprecate chocolatey powershell support and hide it behind a switch. Introduce support for using the Choco vNext library.

**Comment by gep13** [comment]
For this next release, we are simply going to integrate with the Choco.exe.  Then, once the choco.dll is ready, we will replace the direct calls to the exe.

**Comment by RichiCoder1** [comment]
Are we sure we want to close this issue? While we work with Choco vNext, we don't work the way we eventually intend too. Or do we want to make that a seperate issue?

**Comment by gep13** [comment]
I was planning on making a separate issue, to hold the work required to get ChocolateyGUI to work with the Chocolatey DLL, this particular issue has morphed into just getting ChocolateyGUI to work with Chocolatey vNext in the quickest way possible :-)
